projects/vdk-plugins/vdk-dag/src/vdk/plugin/dag/cached_data_job_executor.py

```diff
@@ -138,6 +141,19 @@ def status(self, job_name: str) -> str:
         log.debug(f"Job status: {job}")
         return job.status
 
+    def execution_type(self, job_name: str, team_name: str, execution_id: str) -> str:
+        """
+        Gets the execution type of a job.
+
+        :param execution_id: the execution id of the job
+        :param team_name: the name of the owning team
+        :param job_name: the name of the job
+        :return: the job execution type (manual/scheduled)
+        """
+        details = self._executor.details_job(job_name, team_name, execution_id)
+        log.debug(f"Job execution type: {details.get('type')}")
+        return details.get("type", "manual")
```

Comment:
leave a comment mentioning that we default to "manual" for clarity

Reply:
Done.